AlfrescoApp/Constants/Constants.m

```diff
@@ -84,6 +84,7 @@
 NSString * const kAlfrescoNodeAddedOnServerContentLocationLocally = @"AlfrescoNodeAddedOnServerContentLocationLocally";
 NSString * const kAlfrescoWorkflowTaskListDidChangeNotification = @"AlfrescoWorkflowTaskListDidChange";
 NSString * const kAlfrescoDocumentEditedNotification = @"AlfrescoDocumentEditedNotification";
+NSString * const kAlfrescoURLHandlingEndedNotification = @"AlfrescoURLHandlingEndedNotification";
```

Comment:
Yes, I did struggle to find a name that would better describe its purpose, but as I was focused on the URL handling it seems I lost the bigger picture. Will change it.